runtime/devnet/src/config/xcm.rs

```diff
@@ -28,7 +28,8 @@ use xcm_executor::XcmExecutor;
 parameter_types! {
 	pub const RelayLocation: Location = Location::parent();
 	pub AssetHub: Location = Location::new(1, [Parachain(1000)]);
-	pub const RelayNetwork: Option<NetworkId> = None;
```

Comment:
Do we know why this was `None`?

Reply:
Template: https://github.com/paritytech/polkadot-sdk/blob/10b8039a53ddaa10ca37fe70c64e549c649a22dc/templates/parachain/runtime/src/configs/xcm_config.rs#L27